pyhms/demes/single_pop_eas/sea.py

```diff
@@ -169,3 +171,41 @@ def create(self, **kwargs) -> "GAStyleSEA":
             ],
             k_elites=k_elites,
         )
+
+
+class MWEA(BaseSEA):
+    def __init__(
+        self,
+        variational_operators_pipeline: list[VariationalOperator],
+    ) -> None:
+        self.variational_operators_pipeline = variational_operators_pipeline
+
+    def run(self, parents: list[Individual]) -> list[Individual]:
+        parent_population = Population.from_individuals(parents)
+        offspring_population = parent_population.copy()
+        for variational_operator in self.variational_operators_pipeline:
+            offspring_population = variational_operator(offspring_population)
+        return offspring_population.to_individuals()
+
+    @classmethod
+    def create(self, **kwargs) -> "MWEA":
+        problem = kwargs.get("problem")
+        mutation_std = kwargs.get("mutation_std", DEFAULT_MUTATION_STD)
+        p_mutation = kwargs.get("p_mutation", DEFAULT_P_MUTATION)
+        k_elites = kwargs.get("k_elites", DEFAULT_K_ELITES)
+        election_group_size = kwargs.get("election_group_size", DEFAULT_ELECTION_GROUP_SIZE)
+        return MWEA(
+            variational_operators_pipeline=[
+                MultiwinnerRepeatedSelection(
+                    utility_function=UtilityFunction(
+                        distance=lambda x, y: np.sum(np.abs(x - y)),
```

Comment:
In the paper, they wrote, that any metric can be used. Yet, I'm not sure if Euclid would not be a better choice than Manhattan distance.

Reply:
I wanted to use a metric that works better for highly dimensional spaces.